lighthouse-viewer/app/index.html

```diff
@@ -11,7 +11,7 @@
+  <link rel="icon" href="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAACAAAAAgCAYAAAEEfUpiAAAAAXNSR0IArs4c6QAAB0hJREFUWAmNV2lsFUUcn9l9R33bYkuJHAJWaaG1PKwFapAYwRjiwdECylFj/OrxxcR4gkEkJgY+GEPiB6IfCAGxXgGPaExUNNGIJbYKrVqtCkS52tL29R27O+Pv/9+3+962r+gkuzPzv8+ZXSHyY6x96VF/LUY3NbXr4R2aABF6DTv6RfFQBy29cX5dg1a9ncdGV89jKh9emM/vva9zdGOSsQaBM5/1NgvXKVBcvnxZj66pYwrpgzNbm+pGHOMXqzKeUkNjVvlbXYxjEUSUduQvjusKNTBqCcyjbQ0FHUSQsR1hbt933OroFtE72o5L136B4IGKi2vna1dpYUkwKiWEVqL8oz8CPBGLS+2LZvOi6BVQDG1ZotOOEhWGEobWPyfe6qonusBI23FFbM9dwtrXCg1qgS+EJQzf36Sz0J9xlagyYAP0S6Xesd7v3RioOLM+qeGdqDYQMa1F+fu9jAtUSGCt1z8X1run4AUo84MJLq+dVxuRQpmmKVLr5rOEEIErYwekYxuZB1q8GJCu/Ahs8AH+PLh56dc5pZaXmyRQi6h2kvHD3T/5eH+eICC9tbk9reQBNaVsyNy+orLy6pjQw9mescc/aGDn3v4xxBM46UvMOa60YWJu4Yx+G/mFfiErog1e+jj/PinPIWkjm5q0jWyRh8TqIPZe3ImR4u89YOIckISQBRWHf5AZMGWRMMqsRtEY1dPPWYc6f5WRaJqsiGqxiBJIzCXHudYb9d+tjXpoaEgPDAwgft5QvT8coy5IranrKmbkvvMBF1vrn1HwP1q3sC/z8KppaniwMsU2sum3ER2sWuTT0xyKwQBqXsMFBV/JXQW/y6n+acODgEpYMXOGPNJ/Lg+cfLqwqeXDsfZbJvRIMUfIAh+R3bK4fkTLHhvWVKD64cW3iTdPLPPxxfMEASNbl3TbWiQpha4VS1WkMha1D42I1vVlHd0/FwsIBZEQ6PukizTGnl/V606J1FtVMZF+6tNBdXGkSim3GSQhAaE6IAGKgkiLqkg9CaICMptn9RUCScjCmCiAMgA8Mct8BiRxc1ogfdwIYjC6peVl18096Zcw9QPqUEzFIVTQDn4IKn+vJ+ALYoADQTukJf9IlAiVclDsFEjCjRuBJB9OZxPRcjHBmWlmXigIpNZ96IM6n5bmUAzOr73xZklu0qkHZjCIskd2foVzDJFFc2lVW8xM68AFRkixkZiic+r+jO18YxrOV8usruYeYPzEGIYtkKZcR5ojj+28IKW08DAfv0r4T/CQC3C80YD50ZnXJYnZkHIwu+3BU6m1+ZMWuMy9c28oSB3vAkUdmuwPD5ywD726TLmqKmqKKgC5C4nR0eYGTLt9ISELJAUPj3vwlWW4OjiQ3Ad5wRxcKdp8ZpoDAYOtNZUUQANuUyYMaKXHrwu2AnjcP6GuLIpSsdzJ1zjg5ODmxY8qae5C4V1N7Y4xjOtjW/zg93sROwZMLiGM+V8GfL5iRaR5xuhzwjCedrQqo4Z1KazwaAo088CaBhKXQ73tTih7p+w4mWPgFV7/aQB9OLlG5AApQ4rpuOGoypV1Pea9tbOmVsaFc6T3r+zRk0l0f1HIYYxytybeOXnoCvrHNUIJSpcOBWnCYxhAxXB73Yl464JmRKGB94h4ZN38JB6RO3zquP1Jz9ISYiYFhTuxBBl9cdhQTBGmRkSKUaCoVBx3qAdweKFngtC6hLASoJIpGHno1s0iPbaJvNbSmI2wL/FTIKZa59TCmWfJCDJoennERfezFW7n6bnuhdQM1gOQYYjvMJ2hPTrjm8R7p/aMt6FkBKI5p8kWopWPVEiQ/EFG+UUULo5MV1+MTCeVZL3tdYGXe5bOtvBKuRrfY6Ilb18BwVjvVdIAtNJ+fM91mYaBkjaetbVsJG4KOc3kOd3/mjqOY0hQWhCC1rSkM0F24xR+yQMYp705/Gb2MKiw03fXxv9JJFLIN90Knux8pXumSFFtescfcRk1C/pVf+/1LAEGgNRNZF1LftyXZViJV8kI+HSX4rH78Hlv0gmX94+jIOP4oVi5vid654a4VVvfCHr/RLw+/cSG31TfT/NIBtjMtKlbsTxM+1Ljiga4ptFmOJ6v9I48suubWMvKZUiFhWdJKYHRVZvPZn99jg3gmjHM9aCb1ADf8gmy9I4dBpJ9D779+YLA34mI37z8Grqm+KpCiYeGnet3vjxyzOl4rZZzBXrEALzuGpYVIi5sJo3Apa5DawxllFHhUfiNa2tOm1cl5qH9cs75sz/aR/dLeexoMqXsqCdOU+6D/DMMvOC/aqxz/z3Yf+DRhd+TGgDlG1y+HskPHMFnfp+T2kIfdiIGuYvJQU0FyX9eRFE0GOnDMCtNV2hJA8bFsSAEsldT2CkFyAXOdbQVZHkFSS1G+0IHBGGn45rgbLZnBD7PQnd4QUuheothYrCt8Sb8ZFfRfS4hix9SyEpJMeWWnrwBrBTKeJ+H+VFgnKhKr65ZEVKS3/wLPyrguWhflK8AAAAASUVORK5CYII=">
```

Comment:
gotta run it through imageoptim otherwise lighthouse would be unhappy with it (it wouldnt, but hey)  😉 


```suggestion
  <link rel="icon" href="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAACAAAAAgCAYAAABzenr0AAAEhklEQVR4AWJxL/BhIAesev1U5tcflpncgNrKIsqNIwzC9feMpDUzs70kOczMzMzJJcxwCTMzncPMnOwtzBwzMzPb0vRfeZPp0VhPS5I39V5fdiXV1/VD+9QC7OVn9BsyH1XIoEI1PfmJvLFowVV564+34DFUHudbmfDh4kVXh//7XwE+WjS/YfXZe3yr4j2rqj1AIhSB7hZ8ZtPZu/zw8cK523U4wE1/rvPfWrz4zs0m9ZdC9yUJAlASdBAgocRegfF/f3/h/PuaFsxMdwjAR0vm1+06eMMfIrhLqTWqdH4EumU2SPfMhigJAlRQbZrgrRsl9U+Y2DYDFCz3ILC9kiAiqSrMwbWT0nceEnR+9Kggc2zjOJCASDENkg0a5HfZZgDP81CM3CrQs2Z1+o7DJ6ePr8sK0AOCHv5Jjdt3evyYSaZ351VIStIxPRAUtrBYbxC6w+BZ0ivVSBKkIhJhemSyZpfB00EiPO2VjzYkxhcqXQqCWCShGplvi3y0QxqbuBurMjyJeWnkHZuAEgIQGsUBqwrfjZ+IlBgKyRJzVVYF8O6qFWdh86YzQzMrZigYmxAyfvHgLZQ/LC1CbeniW2Hkqr/PH16SgvGuf2/uzNMBwJA/njxizGPtSyAf7EziJCMGRDRdhoAC4PL1A/SrKQMAAQkEfpJAcRQdrBJ7gNwjSpJsdwK+CANBkqa1LgQB4IicV9nYUct7gaxuDJUErQIiEAiMxLVOFlKzIktPpT0ggpdpC/8YAHnxbgkUY4tAAFSR7AAXNyAAWHJrA/kHGjzg5nleuwFO7Nd/IoDw4Pm58+4jNLmYG0wRA5bErc2Mr3Y+dXTDW1VvwqbJkzMCHQ4S1GTCBOIgUHJrGdEwqzR+jAp/o2qAZelUDoQnruEEdDclJI6576AlNVfc+22XN/+Y1vnJD0Yind6UpEEvn/Hqq15EYjCW7jZCJEpnNvDgkyelDjs106kuux2AAXCSobULOWP8mLhYlpoDMK4qAFXJGk+grtH8YXVz5KJblqaG1+VUdTc0I290bmUQAriGITRbdQnom0aoFj8kx1+wMD2ifncAXUQE4SkDqN1hE0jEophs1SUwZAOhUAiMCLwRtamtTZtbbmZErSAUHbSysaoEmnrsakiMiUAURi283gN6wans9oX8rOCrj7/JP35DFD+iQ7Au/K2KE1jzx6ujjUnXFH9KjEq6ZlhsTBICrNLJf47Pv/pkHzvup1w4dmUbEei0+bcXRqJuh5kVARQ8byyYxOwNGr7A87xh1tp8sGT+uMInrwi++Xj7TQz2d27NvwEkrOflAFQGIDA5khASBCGdO2/Z/MnLPwYfv5TFhjW7QhVKAB6afwe2LpFlFsCnlQEosgQgDsdOG1/LKeNqJS4JCSPJ/i+TakwEARor7gER1Iva5JmPOJK0RUqmoPnnlzFCtmIAhAAQEIQRgDaiYPIauNXcnDlRIrWNFY3hm7PG9YRqr7IV7HrCgAC17befjEvRq2nGhAHtBqDpOuI/I1diUUAMYIxEdyejBJqLnNoszGZtfiX/CztGv2mq+sdaAAAAAElFTkSuQmCC">
```